src/main/java/com/zendesk/maxwell/producer/MaxwellRedisProducer.java

```diff
@@ -0,0 +1,74 @@
+package com.zendesk.maxwell.producer;
+
+import com.zendesk.maxwell.MaxwellContext;
+import com.zendesk.maxwell.row.RowMap;
+import com.zendesk.maxwell.util.StoppableTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+public class MaxwellRedisProducer extends AbstractProducer implements StoppableTask {
+	private static final Logger logger = LoggerFactory.getLogger(MaxwellRedisProducer.class);
+	private final String channel;
+	private final Jedis jedis;
+
+	public MaxwellRedisProducer(MaxwellContext context, String redisPubChannel) {
+		super(context);
+
+		channel = redisPubChannel;
+
+		jedis = new Jedis(context.getConfig().redisHost, context.getConfig().redisPort);
+		jedis.connect();
+		if (context.getConfig().redisAuth != null) {
+			jedis.auth(context.getConfig().redisAuth);
+		}
+		if (context.getConfig().redisDatabase > 0) {
+			jedis.select(context.getConfig().redisDatabase);
+		}
+	}
+
+	@Override
+	public void push(RowMap r) throws Exception {
+		if ( !r.shouldOutput(outputConfig) ) {
+			context.setPosition(r.getPosition());
+			return;
+		}
+
+		String msg = r.toJSON(outputConfig);
+		try {
+			jedis.publish(this.channel, msg);
+			this.succeededMessageCount.inc();
+			this.succeededMessageMeter.mark();
+		}
+		catch (Exception e) {
```

Comment:
nit: will be nicer to combine the above 2 lines into one line: `} catch (...) }`

Reply:
👍 oops. JS style habits 😄 
fixed